sdk/@launchdarkly/observability-node/src/index.ts

```diff
@@ -1,3 +1,34 @@
+/**
```

Comment:
Shows in the index of the API docs.
<img width="719" height="567" alt="image" src="https://github.com/user-attachments/assets/3e0345c9-2edb-4c4c-89f2-537883abe8a9" />

Reply:
neat